pkg/trace/stats/client_stats_aggregator.go

```diff
@@ -182,7 +186,21 @@ type bucket struct {
 	agg map[PayloadAggregationKey]map[BucketsAggregationKey]*aggregatedCounts
 }
 
-func (b *bucket) add(p *pb.ClientStatsPayload, enablePeerSvcAgg bool) []*pb.ClientStatsPayload {
+// FIXME: resolve import cycle so that we don't need to duplicate this
+func getImageTag(containerID string, conf *config.AgentConfig) (string, error) {
```

Comment:
This is super annoying. Would appreciate a suggestion on how we can break the import cycle - maybe I'll just need to put this function into its own package (but not `traceutil`, since that will cause a cyclic import of its own).